src/css/styles.scss

```diff
@@ -583,6 +583,20 @@
       padding: 0;
     }
   }
+
+  .excalidraw-button {
+    @include outlineButtonStyles;
+    background-color: var(--island-bg-color);
+    width: var(--lg-button-size);
+    height: var(--lg-button-size);
+
+    svg {
+      width: var(--lg-icon-size);
+      height: var(--lg-icon-size);
+    }
+
+    overflow: hidden;
+  }
```

Comment:
one trouble is it's still nested (?) in the `.excalidraw` namespace so using it outside the editor won't work :p.

I'd put it outside since the class is already namespaced by way of its name!

Reply:
discussed on chat. We'll want to do this instead https://github.com/excalidraw/excalidraw/issues/6089 because of the css variables and dark mode